.ci/Jenkinsfile

```
@@ -344,15 +344,26 @@ def scpr(workspace, connection, remote_src, local_dst){
   }
 }
 
+def getPythonPath(version) {
+  sh(script: "curl https://pyenv.run | bash", label: "Install pyenv")
```

Avoid failures when the `.pyenv` folder exists:
```suggestion
  sh(script: "curl https://pyenv.run | bash", label: "Install pyenv", returnStatus: true)
```